ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3RemoteServiceAdd.java

```diff
@@ -77,6 +77,14 @@ protected void performBoottime(OperationContext context, ModelNode operation, Mo
         installRuntimeServices(context, model);
     }
 
+    @Override
+    protected void populateModel(ModelNode operation, ModelNode model) throws OperationFailedException {
+        EJB3RemoteResourceDefinition.CLIENT_MAPPINGS_CLUSTER_NAME.validateAndSet(operation, model);
+        EJB3RemoteResourceDefinition.CONNECTOR_REF.validateAndSet(operation, model);
+        EJB3RemoteResourceDefinition.THREAD_POOL_NAME.validateAndSet(operation, model);
+        EJB3RemoteResourceDefinition.EXECUTE_IN_WORKER.validateAndSet(operation, model);
+    }
```

Comment:
I think this move introduces merge conflicts because the whole method is gone now.

Reply:
Not sure I understand. When I open the file in the PR, I see the method. Do you mean it disappears after being merged?

Reply:
When I wrote that comment Github was reporting this PR had a merge conflict.  At the time, github was showing the change this PR was making to this file was moving this method from lower in the file to here. I assumed that was the merge conflict because Paul had done a different change to this file that removed this method altogether.

It looks like now the conflict has been resolved but the resolution resulted in restoring this method. I don't think it should be restored. Give the rest of Paul's change all it's doing here is what the superclass implementation will do.

Reply:
Removed the method populateModel() as it is no longer in master..